### DISCUSSION TITLE
Shape Size is not adapting if the text size becomes longer than shape's dimensions

### DISCUSSION THREAD

**Discussion by vyombriq** [clientIO/joint#2801] [Q&A]
### Introduction

If text content on dropped shape exceeds shape dimensions then the shape size is not dynamically increasing, is there a way we can handle that if text content exceeds the height of the shape it should increase shape's size automatically 

### Restrictions & Constraints

i am using standard shapes and need solution with those 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Here are examples of how to automatically resize a shape based on the content it contains:
- https://www.jointjs.com/demos/content-driven-shapes
- https://www.jointjs.com/demos/rich-text-editor
- https://www.jointjs.com/demos/dialog-generator
- https://www.jointjs.com/demos/mindmap
